grpc_server.go

```diff
@@ -740,3 +741,50 @@ func (s *server) RealTime(stream pb.SpaceCloud_RealTimeServer) error {
 	})
 	return nil
 }
+
+func (s *server) Profile(ctx context.Context, in *pb.ProfileRequest) (*pb.Response, error) {
+	status, err, result := s.user.Profile(ctx, in.Meta.Token, in.Meta.DbType, in.Meta.Project, in.Id)
+	resp := generateResponse(status, err, result)
+	return resp, nil
+}
+
+func (s *server) Profiles(ctx context.Context, in *pb.ProfilesRequest) (*pb.Response, error) {
+	status, err, result := s.user.Profiles(ctx, in.Meta.Token, in.Meta.DbType, in.Meta.Project)
+	resp := generateResponse(status, err, result)
+	return resp, nil
+}
+
+func (s *server) EditProfile(ctx context.Context, in *pb.EditProfileRequest) (*pb.Response, error) {
+	status, err, result := s.user.EmailEditProfile(ctx, in.Meta.Token, in.Meta.DbType, in.Meta.Project, in.Id, in.Email, in.Name, in.Password)
+	resp := generateResponse(status, err, result)
+	return resp, nil
+}
+
+func (s *server) SignIn(ctx context.Context, in *pb.SignInRequest) (*pb.Response, error) {
+	status, err, result := s.user.EmailSignIn(ctx, in.Meta.DbType, in.Meta.Project, in.Email, in.Password)
```

Comment:
Err must me the last parameter according to golang conventions

Reply:
Ok